Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM ubuntu:18.04
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV DEBIAN_FRONTEND noninteractive
+ENV RENV_VERSION 0.8.2
+ENV RENV_PATHS_CACHE /renv/cache
+ENV GOOGLE_APPLICATION_CREDENTIALS /mc-etl/gcloud.json
```

Comment:
How is this getting injected? It's probably worth adding a comment here to discuss expected deployment constraints.

Reply:
Yeah, definitely need to think this part through. I can't remember if we settled on a best practice for this?

Reply:
This will change before we deploy in airflow for sure. Resolving for now.